src/main/java/com/tfgm/controllers/TramController.java

```diff
@@ -58,13 +58,13 @@ public TramController(String tramDataPath) throws IOException {
               .toArray(TramStopContainer[]::new));
     }
 
-    for (NewTramStop newTramStop : tramStopHashMap.values()) {
+    for (TramStop tramStop : tramStopHashMap.values()) {
 
-      System.out.println(newTramStop.toString());
-      for (TramStopContainer n : newTramStop.getNextStops()) {
+      System.out.println(tramStop.toString());
```

Comment:
I'd recommend using a logger instead of System.out before you get too far into the project